### ISSUE TITLE
Texture themes

### ISSUE THREAD

**Issue by wtfrazr** [Pottus/Texture-Studio#76]
Hi again.
I'm sure I am getting annoying, but can it be made that the texture themes save just like a map saves?
I mean if you have an open theme it will automatically save as you update it, and it will still be open after you leave the game as a client but the server is still running.
It is annoying to lose my saved textures sometimes when I forget to save, when I need to re-log.

**Comment by Crayder** [comment]
Good idea, especially now that I have made the 3d viewer a lil bit better.
